engine/lib/users.php

```diff
@@ -449,10 +449,16 @@ function user_avatar_hook($hook, $entity_type, $returnvalue, $params) {
 	$size = $params['size'];
 
 	if (isset($user->icontime)) {
-		return "avatar/view/$user->username/$size/$user->icontime";
```

Comment:
This particular view is not really extendable, it sends headers so if you output content before or after, you would be making a mess. We could just check view location. Trying that now.